test/unit/text.js

```diff
@@ -1,6 +1,13 @@
 (function() {
 
-  QUnit.module('fabric.Text');
+  QUnit.module('fabric.Text', {
+    before() {
+      fabric.config.configure({ NUM_FRACTION_DIGITS: 2 });
```

Comment:
I have no idea why this is needed
w/o tests fail
I will inspect

Reply:
all the tests are usually done with NUM_FRACTION_DIGITS that was the old defaults to avoid large exports and approximation error between js implementations

Reply:
I pointed this out since it was I change I made to tests for them to pass, before they were passing without `NUM_FRACTION_DIGITS` assignment, not sure if it was pollution or what